utils/db.ts

```diff
@@ -387,39 +394,90 @@ export function compareScore(a: Item, b: Item) {
 }
 
 // Analytics
-export async function incrementAnalyticsMetricPerDay(
-  metric: string,
-  date: Date,
-) {
+export async function incrementVisitsCountByDay(date: Date) {
```

Comment:
```suggestion
export async function incrVisitsCountByDay(date: Date) {
```
Nit. WDYT? Is it still clear to a newcomer what this does?

Reply:
Makes sense. I think it is still clear!